src/praisonai/praisonai/bots/_commands.py

```
@@ -495,6 +495,64 @@ def handle_run_status_command(
+        )
+
+        boot_fp = getattr(session_manager, "_boot_code_fp", None)
+        if boot_fp is None:
+            # First call without a captured boot fingerprint: capture it now so
+            # this and subsequent checks have a baseline. This still catches a
+            # later in-place update; it just cannot guard the very first call.
+            boot_fp = read_code_fingerprint()
+            try:
+                session_manager._boot_code_fp = boot_fp
+            except Exception:
+                pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)